apps/dev-portal-web/docs/api/examples/auth.en.mdx

```diff
@@ -43,6 +45,37 @@ curl 'https://condo.d.doma.ai/admin/api' -H 'Content-Type: application/json' --d
 ```
 </Tabs>
 
+## authenticateUserWithPassword
```

Comment:
it's better to suggest new auth mutations....

Reply:
looks like them not ready yet